app/views/errors/not_found.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>404 - <%= t('not_found.title') %> - <%= t('cm-central') %></title>
+  <%= csrf_meta_tag %>
+  <%= favicon_link_tag "favicon.ico" %>
+  <%= stylesheet_link_tag :application %>
+</head>
+<body class="NotFound">
+  <div class="NotFound__container">
+    <h2 class="NotFound__title">404</h2>
+    <h3 class="NotFound__description"><%= t('not_found.description') %></h3>
+    <%= link_to t('back'), root_path, class: "NotFound__link" %>
```

Comment:
I think we're missing the back reference here (and then fallback to `root_path`):

```suggestion
    <%= link_to t('back'), request.referer || root_path, class: "NotFound__link" %>
```

Reply:
Oh thanks!